This needs some more work:

```
sh: line 1: node-pre-gyp: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! sqlite3@5.0.2 install: `node-pre-gyp install --fallback-to-build`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the sqlite3@5.0.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /build/.npm/_logs/2022-05-14T06_15_45_978Z-debug.log
```

--- a/pkgs/applications/video/epgstation/default.nix
+++ b/pkgs/applications/video/epgstation/default.nix
@@ -41,7 +41,7 @@ let
     # dependencies are pruned afterwards.
     production = false;
 
-    buildInputs = [ bash ];
+    buildInputs = (drv.buildInputs or [ ]) ++ [ bash ];
     nativeBuildInputs = (drv.nativeBuildInputs or [ ]) ++ [
       makeWrapper
     ];

